test/nerves_time/rtc/ds3231/status_test.exs

```diff
@@ -2,9 +2,17 @@ defmodule NervesTime.RTC.DS3231.StatusTest do
   use ExUnit.Case
   alias NervesTime.RTC.DS3231.Status
 
-  test "decodes Status" do
-    assert Status.decode(<<0xFF>>) ==
-             {:ok,
-              %{alarm_1_flag: 1, alarm_2_flag: 1, busy: 1, ena_32khz_out: 1, osc_stop_flag: 1}}
+  test "decode/1 and encode/1" do
+    data = %{
+      alarm_1_flag: 1,
+      alarm_2_flag: 1,
+      busy: 1,
+      ena_32khz_out: 1,
+      osc_stop_flag: 1
+    }
+
+    bin = <<0x8F>>
```

Comment:
Just pointing out that this changed from an `0xFF` above because 3 bits get correctly thrown away when decoding. In other words, previously `Status.decode(<<0xFF>>)` would yield `data` above. But so would all of these `["8F", "9F", "AF", "BF", "CF", "DF", "EF", "FF"]`. So now that we are also _encoding_ it does matter, because `<<0b1111_1111>>` encoded and then decoded would be.... `<<0x8F>>`.

TLDR: `Status.decode/1` is (and was) lossy, but only on bits with no meaning.